### DISCUSSION TITLE
Home + Warp bungee network sync

### DISCUSSION THREAD

**Discussion by TheoRetisch1** [Zrips/CMI#4718] [Feature Requests]
It would be fantastic if CMI would enable the synchronization of homes and warps across multiple servers in a bungee network. There have been open feature requests and countless requests for this feature in the community discord for years.

**With hundreds of open feature requests, why would you bother with this complex feature?** Well it concerns the core competence of the core plugin CMI. Everybody uses and needs warps and homes, but many servers do not benefit from the CMI features because they depend on bungeecord. For the majority it would create a much greater value than any other feature. At this point in time it' s worth going back to the roots and give CMI a new state-of-the-art feature in the core area.

**Comment by BearLeonLight** [comment]
If possible, I hope this feature request can also be added to TPA、TPAHERE

**Comment by rodrigoveglo** [comment]
+1

**Comment by mrfloris** [comment]
I would love for homes and warps to sync over bungee and velocity, it's a highly requested feature. same with tablist
